test/powershell/Modules/Microsoft.PowerShell.Core/Job.Tests.ps1

```diff
@@ -9,9 +9,22 @@ Describe "Job Cmdlet Tests" -Tag "CI" {
         It "Start-Job produces a job object" {
```

Comment:
We can certainly ad the `-Name` parameter in the `BeforeEach` block and merge the first two test together. Personally, I'd rather keep the `Get-Job` test separate.